src/components/badges/TextBadge.jsx

```diff
@@ -1,22 +1,35 @@
+// @flow
```

Comment:
why not strict?

Reply:
it depends on non-strict module, `Text`, which Yury works on. after he will introduce `flow strict` there, i will be able to cover TextBadge as well. don't worry, status of `flow strict` coverage is tracked here: https://docs.google.com/spreadsheets/d/1uacUj6OAwSeM9kn5IuIdtOfw8QGQNpV1_I3Zqk5SZE0/edit?pli=1#gid=0